src/components/Admin/ManageSensors/DeleteOldDataModal.tsx

```diff
@@ -131,7 +131,10 @@ function DeleteOldDataModal(): JSX.Element {
         mapping[sensorDoc.id] = deleteBeforeDate;
       }
 
-      deletionDocRef.update({deletionMap: mapping});
+      deletionDocRef.update({
+        deletionMap: mapping,
+        lastUpdated: firebase.firestore.FieldValue.serverTimestamp(),
```

Comment:
lastUpdated refers both to the last time new docs were marked for deletion and the last time docs were deleted. What's the point of the field?

Reply:
Similar to why I added it to each of the sensor docs, it's to make sure that some field is updated whenever `.update()` is called, to avoid errors, and then I added it to these fields for consistency. I could also change this to be `lastDeleted` and `mostRecentUpdate` or something like that